content/02.introduction.md

```diff
@@ -1,6 +1,19 @@
-#Introduction
-Set introduction for paper here
-Talk about problem, goal, and significance of paper
+# Introduction
+
```

Comment:
Would start with: why do we care about knowledge bases? Are there some cool examples of how they get used?

Reply:
Addressed by the new sentences 👍